autoload/airline/extensions/wordcount.vim

```diff
@@ -9,8 +9,14 @@ if exists('*wordcount')
     if get(g:, 'actual_curbuf', '') != bufnr('')
       return
     endif
-    let query = a:visual_mode_active ? 'visual_words' : 'words'
-    return get(wordcount(), query, 0)
+    if &filetype ==# 'tex' && exists('b:vimtex')
+" We're in a TeX file and vimtex is a plugin, so use it's wordcount...
+      let value = vimtex#misc#wordcount()
+      return value
```

Comment:
Can you please change this from:
 
```
 let value = vimtex#misc#wordcount()
 return value
```
change it to: 
```
return vimtex#misc#wordcount()
```
That would be all. I believe.